crates/environ/src/component.rs

```diff
@@ -154,6 +154,9 @@ macro_rules! foreach_builtin_component_function {
             #[cfg(feature = "component-model-async")]
             error_context_transfer(vmctx: vmctx, src_idx: u32, src_table: u32, dst_table: u32) -> u64;
 
+            #[cfg(feature = "threads")]
+            thread_spawn_indirect(vmctx: vmctx, table: u32, element: u32, context: u32 ) -> u64;
```

Comment:
For 64-bit WebAssembly, we probably want `element` (and probably even `context`) to be a `u64`; how are we handling the 32-bit/64-bit distinction elsewhere?

Reply:
Currently it's basically not-handled, Wasmtime [doesn't support 64-bit components at all](https://github.com/bytecodealliance/wasmtime/issues/4311)